src/Shared/Telemetry/TelemetryConstants.ts

```diff
@@ -86,7 +86,10 @@ export enum Action {
   CreateMongoCollectionWithWildcardIndex,
   ClickCommandBarButton,
   RefreshResourceTreeMyNotebooks,
-  ClickResourceTreeNodeContextMenuItem
+  ClickResourceTreeNodeContextMenuItem,
+  DiscardSettings,
+  UpdateSettingsV2,
+  DiscardSettingsV2

```

Comment:
>DiscardSettingsV2 [](start = 2, length = 17)

uggest: prefix with area like: SettingsUpdated, SettingsV2Updated, NotebooksRunCell...etc. So we can find all telemetry constants related to an area together when writing them in sorted order in this list.

Reply:
the old action for updating settings was "UpdateSettings", so I followed that pattern. but, sure, will change it for SettingsV2